tests/resource-timing.spec.ts

```diff
@@ -26,10 +26,74 @@ it('should work', async ({ contextFactory, server }) => {
   ]);
   const timing = request.timing();
   verifyConnectionTimingConsistency(timing);
+  expect(timing.startTime).toBeGreaterThanOrEqual(request.issueTime());
   expect(timing.requestStart).toBeGreaterThanOrEqual(timing.connectEnd);
   expect(timing.responseStart).toBeGreaterThanOrEqual(timing.requestStart);
   expect(timing.responseEnd).toBeGreaterThanOrEqual(timing.responseStart);
-  expect(timing.responseEnd).toBeLessThan(10000);
+  await context.close();
+});
+
+it('should account for blocked and queued time', async ({ contextFactory, server }) => {
```

Comment:
So I just ran your test w/o your modifications and I got this for connectStart:

```
0.405
-1
0.703
0.923
0.464
0.526
0.693
206.547
206.497
206.517
206.517
206.651
206.766
414.953
```

- So it looks like as far as blocking time is concerned, we are covered with the existing code, at least on Chromium.
- There is no clear distinction between blocking and queueing time, so we should decide what you want to see there. If you are interested in deferring requests by the renderer based on their priority (scripts go before images), we would need a test page for that. Otherwise all your numbers are basically zeros, a cost of an mojo plumbing calls.

Reply:
Thanks for testing it and the details. 

> So it looks like as far as blocking time is concerned, we are covered with the existing code, at least on Chromium.

What would that look like with the existing code? duration between startTime and dns/connect start? I would like to at-least patch that in the hartracer. 


>  If you are interested in deferring requests by the renderer based on their priority (scripts go before images), we would need a test page for that.

Yes when i was referring to that for the queue time but thought of doing that later on in separate PR.

To conclude if we can get the blocking time now without exposing the issueTime, I am okay with closing down this PR.

Reply:
> What would that look like with the existing code? duration between startTime and dns/connect start? I would like to at-least patch that in the hartracer.

Yes, I ran your test and printed `connectStart`. If you have other tests, I can run those as well. But this particular use case seems to be covered with the existing code.

> Yes when i was referring to that for the queue time but thought of doing that later on in separate PR.

Sure, we would need a different test for that.

> To conclude if we can get the blocking time now without exposing the issueTime, I am okay with closing down this PR.

That's up to you. There still are inconsistencies in other browsers, for example WebKit on a mac was unblocking not when 6 connections were closed, but when they were responded to. But I think this reflects the reality.

Since your changes were somewhat Chromium centric, I don't think they are adding to the blocking time, so yeah feel free to close.

Reply:
Thanks for the details, Will close this PR and will try to incorporate changes to harTracer for blocked time.